src/StoreApi/Routes/Checkout.php

```diff
@@ -95,7 +89,7 @@ public function get_args() {
 										'type' => 'string',
 									],
 									'value' => [
-										'type' => 'string',
+										'type' => [ 'string', 'boolean' ],
```

Comment:
Once I fixed schema validation, I noticed an error from the payments due to a "true" param from a payment method. This prevents it breaking due to schema mismatch.